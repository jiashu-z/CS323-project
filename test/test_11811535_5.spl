int a b;
int func_1(int a int b)
{
  return 3+a*b-2/1;
}
int test_11811535_1){
  int a=1;
}