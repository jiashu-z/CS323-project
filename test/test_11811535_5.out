Error type B at Line 1: Missing comma ','
Error type B at Line 2: Missing comma ','
Error type B at Line 6: Missing left parenthesis ')'